phpBB/config/default/container/services_text_reparser.yml

```diff
@@ -0,0 +1,63 @@
+services:
+    text_reparser.contact_admin_info:
```

Comment:
please also add a service for the reparser plugin collection (text_reparser.lugins_collection maybe) so it can be used